Update troubleshooting document with 400 BadDeviceToken (#158)
matrix-org/sygnal: docs/troubleshooting.md

Firebase with your APNs secrets, and Sygnal with your Firebase secrets.

### App doesn't receive notifications when inactive

If you want your application to be woken up to be able to process APNs messages
received when your application is in the background, you need to set the
`content-available` flag in your pusher's default payload — see
[the notes for iOS applications](applications.md#ios-applications-beware).

### '400 BadDeviceToken' error

> If you have a bad device token and your token was not a fictitious invention,
> the most likely cause is that you have used a token from the wrong 'environment',

Reviewer: ```suggestion
If you get a bad device token error the an possible cause is that
you have used a token from the wrong 'environment',
```

Reviewer 2: > error the an possible

extra "an"

> If you have a bad device token and your token was not a fictitious invention,
> the most likely cause is that you have used a token from the wrong 'environment',

Reviewer 2: ```suggestion
If you get a bad device token error and you have doubled-checked the
token is correct, it is possible that you have used a token from the wrong 'environment',
```
